bottlecap/src/config/mod.rs

```diff
@@ -50,7 +50,7 @@ fn fallback(config: &EnvConfig, yaml_config: &YamlConfig) -> Result<(), ConfigEr
         ));
     }
 
-    if config.serverless_appsec_enabled || config.appsec_enabled {
+    if config.serverless_appsec_enabled {
         log_fallback_reason("appsec_enabled");
```

Comment:
Probably wanna update the error message here to reflect the serverless appsec enablement being the cause.